workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandResponseHandler.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using Improbable.Worker.Core;
+using Unity.Entities;
+using Entity = Unity.Entities.Entity;
+
+// ReSharper disable EventNeverSubscribedTo.Global
+// ReSharper disable UnusedAutoPropertyAccessor.Global
+// ReSharper disable MemberCanBePrivate.Global
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static partial class WorldCommands
+    {
+        public struct CreateEntityResponse
```

Comment:
I'm getting confused by where this is used, when are we using something in Core and when are we using these etc. Can this also be removed if we use the InjectableStore as mentioned above? (I really don't know based on a quick look...)